inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd

```diff
@@ -73,6 +79,12 @@ The {.unnumbered} option here means that the introduction will be
 "Chapter 0." You can also use {-} for no numbers on chapters.
 -->
 
+```{r, include=FALSE}
+knitr::write_bib(c(.packages(), "bookdown"), "bib/packages.bib")
+```
+
+<!-- This chunk creates automatically a .bib file to cite the loaded packages in the bibliography. Do not forget to uncomment "nocite: '@*'" in the header if you want all packages to be cited even if you did not mention them in the body of the text.-->
```

Comment:
Put the instructions above or within so that users don't accidentally delete or get turned off as they read from top to bottom